pkg/installer/custominstallconfig_test.go

```diff
@@ -271,3 +305,119 @@ func TestApplyInstallConfigCustomisations(t *testing.T) {
 
 	t.Log(bootstrapIgnition)
 }
+
+func TestApplyManifestsCustomization(t *testing.T) {
+	m := fakeManager()
+	inInstallConfig := makeInstallConfig()
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	mockClient := mock.NewMockAPI(mockCtrl)
+	inInstallConfig.Azure.UseMockClient(mockClient)
+	mockClientCalls(mockClient)
+
+	g, err := m.applyInstallConfigCustomisations(inInstallConfig, makeImage())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, a := range targets.Manifests {
+		err := g.Resolve(a)
```

Comment:
I understand this is meant to emulate the code in the `create manifests` command.

What I don't understand is why _that_ code exists. Given that `applyInstallConfigCustomisations()` already resolves all of the assets up to `targets.Cluster`, and all of the assets in `targets.Manifests` are in the dependency tree for `targets.Cluster`, they have been generated anyway and `Resolve()` will just return nil immediately, so what is the point of any of it?

The test works exactly the same without any of this, so I think we should call `verifyIgntionFiles()` directly from `TestApplyInstallConfigCustomisations()` and not create `TestApplyManifestsCustomization()` as a duplication of it.

Reply:
Was looking into the future on this one since we need to have customisations for each stage but I'll remove this.

Reply:
If it makes you feel better, that is not something in our future :slightly_smiling_face: 
We're going to do all the customisation inside `applyInstallConfigCustomisations()` the same as it is now, because that is the thing that we can unit test.